webpack/webpack.config.base.js

```diff
@@ -62,15 +62,15 @@ module.exports = function getBaseConfig(config) {
           loader: 'url-loader',
```

Comment:
Can you check, why the filename is defined in this file as `'[name].js',`, but rewritten in the client config with config.filename. Is that necessary?